### PR TITLE
Migrate localization to weblate

### PR DESCRIPTION
**Pull request by jkonecny12**
Switch our workflow from Zanata to Weblate.

The main changes are in the Makefile.

We have disabled automatic pot file push during the release because we want to be cautious for now. In the future we can create automation to push pot file after each merge.

**Warning:**
Some parts of this PR is hard to test. So makebumpver.py is not really tested and it should be. Could you please @M4rtinK look on that if the script is working correctly after this change?

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -99,7 +99,33 @@ po-pull:
 po-push:
 	$(MAKE) -C po $(PACKAGE_NAME).pot-update
 	rm $(srcdir)/po/{main,extra}.pot
-	$(ZANATA_CLIENT_BIN) push $(ZANATA_PUSH_ARGS)
+# This algorithm will make these steps:
+# - clone localization repository
+# - copy pot file to this repository
+# - check if pot file is changed (ignore the POT-Creation-Date otherwise it's always changed)
+# - it does not changed:
```
Comment (M4rtinK) [inline Makefile.am]: I think this should be:
"if not changed"

File: Makefile.am [hunk 2]
```diff
@@ -99,7 +99,33 @@ po-pull:
...
+#   - remove cloned repository
+# - it does changed:
```
Comment (M4rtinK) [inline Makefile.am]: "if changed"

File: Makefile.am [hunk 3]
```diff
@@ -99,7 +99,33 @@ po-pull:
...
+#   - add pot file
+#   - commit pot file
+#   - tell user to verify this file and push it in the temp dir
```
Comment (M4rtinK) [inline Makefile.am]: "push it to the remote from the temp dir" ?

File: Makefile.am [hunk 4]
```diff
@@ -180,10 +175,8 @@ bumpver: po-pull
 	if [ ! -z "$(SKIP_ACKS)" ]; then \
 		opts="$${opts} -s" ; \
 	fi ; \
-	( cd $(srcdir) && scripts/makebumpver $${opts} ) || exit 1 ; \
-	$(MAKE) -C po $(PACKAGE_NAME).pot-update && \
-	rm $(srcdir)/po/{main,extra}.pot
-	$(ZANATA_CLIENT_BIN) push $(ZANATA_PUSH_ARGS)
+	( cd $(srcdir) && scripts/makebumpver $${opts} ) || exit 1 ;
```
Comment (rvykydal) [inline Makefile.am]: I wonder if there should bre '\' at the and of the line. Perhaps not necessary, but there are places we do that, so I better ask.

File: Makefile.am (hunk 1 above)
Comment (jkonecny12) [reply Makefile.am]: Fixed.

File: Makefile.am (hunk 2 above)
Comment (jkonecny12) [reply Makefile.am]: Fixed.

File: Makefile.am (hunk 3 above)
Comment (jkonecny12) [reply Makefile.am]: Sounds much better. Fixed.

File: Makefile.am (hunk 4 above)
Comment (jkonecny12) [reply Makefile.am]: It's correct. We don't share anything from the above in the last line so there is no reason to have it as one shell.
However, I removed have `;` on the last line. :)

**Comment by jkonecny12** [comment]
**UPDATED:**
* Fix grammar issues
* Fix builds (removing *.txt from a make dist)
* Remove unneeded semicolon from Makefile

**Comment by jkonecny12** [comment]
This PR was closed by accident and can't be reopened...

**Pull request closed without merging**